istio-telemetry/kiali-operator/templates/deployment.yaml

```diff
@@ -0,0 +1,87 @@
+{{- if .Values.kialiOperator.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kiali-operator
+  namespace: {{ .Values.kialiOperator.namespace }}
+  labels:
+    app: kiali-operator
+    release: {{ .Release.Name }}
+spec:
+  replicas: {{ .Values.kialiOperator.replicaCount }}
+  selector:
+    matchLabels:
+      app: kiali-operator
+  template:
+    metadata:
+      name: kiali-operator
+      labels:
+        app: kiali-operator
+        release: {{ .Release.Name }}
+      annotations:
+        sidecar.istio.io/inject: "false"
+        scheduler.alpha.kubernetes.io/critical-pod: ""
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "8383"
```

Comment:
should this port not be `42422`?

Reply:
No, it is 8383. This is the metrics port that base ansible operator SDK opens up and exposes its metrics -- see https://github.com/operator-framework/operator-sdk/blob/v0.9.x/pkg/ansible/run.go#L43

So these are operator metrics (the operator itself exposes its own metrics to monitor itself).

See: https://github.com/kiali/kiali/issues/1561 (that issue documents a snippet of the metrics that prometheus will be able to scrape on the operator's port 8383).